Blog/templates/Blog/base.html

```diff
@@ -9,11 +9,31 @@
         <link rel="stylesheet" href="{% static 'Blog/style.css' %}" type="text/css">
     </head>
     <body>
-        <header>
-                <nav class="navbar navbar-light bg-primary" style="">
-                    <span class="navbar-brand mb-0 h1">{% block header %}{% endblock %}</span>
-                </nav>
-        </header>
+        <nav class="navbar navbar-expand-sm bg-dark navbar-dark">
+          <ul class="navbar-nav">
+              <li class="nav-item active col-sm-12" style="width:700px;">
+                <a class="nav-link" href="#">{% block header %}{% endblock %}</a>
+              </li>
+              <li class="nav-item">
+                  <a class="btn btn-link" href="{% url 'Blog:homepage' %}">Home</a>
+              </li>
+              {% if request.user.is_authenticated %}
+                  <li class="nav-item float-right">
+                      <a class="btn btn-link" href="{% url 'Blog:mine' %}">/Mine</a>
+                  </li>
+                  <li class="nav-item float-right">
+                      <a class="btn btn-link" href="{% url 'Blog:logout' %}">LOGOUT</a>
```

Comment:
Logout should happen only via post request. Make a form with submit button that makes a post request.